bin/monitor

```diff
@@ -4,7 +4,7 @@
 require_relative "../loader"
 clover_freeze
 
-MONITORABLE_RESOURCE_TYPES = [PostgresServer, MinioServer, GithubRunner]
+MONITORABLE_RESOURCE_TYPES = [PostgresServer, MinioServer, GithubRunner, VmHost, Vm.left_join(:sshable, id: :id).where(Sequel[:sshable][:id] => nil)]
```

Comment:
I kind of love this idea of resource types (it's not exactly a type in a conventional sense) being dataset-ish.